src/i18n/locales/en-US.json

```diff
@@ -84,9 +84,9 @@
   "components.send.confirmscreen.receiver": "Receiver",
   "components.send.confirmscreen.sendingModalTitle": "Submitting transaction",
   "components.send.confirmscreen.title": "Send",
-  "components.send.sendscreen.addressInputErrorInvalidAddress": "Please enter valid address",
+  "components.send.sendscreen.addressInputErrorInvalidAddress": "Please enter a valid address",
   "components.send.sendscreen.addressInputLabel": "Address",
-  "components.send.sendscreen.amountInput.error.INVALID_AMOUNT": "Please enter valid amount",
+  "components.send.sendscreen.amountInput.error.INVALID_AMOUNT": "Please enter a valid amount",
```

Comment:
there are a few other strings that don't have an "a". If you search for "Please enter" you can find and correct them all

Reply:
You're right. I went and got the rest. 
I left instances of "Enter PIN" as they are since that doesn't need to be changed.